internal/sirius/teams.go

```diff
@@ -15,11 +15,17 @@ type apiTeam struct {
 	} `json:"teamType"`
 }
 
-type Team struct {
-	ID          int
+type TeamMember struct {
 	DisplayName string
-	Members     int
-	Type        string
+	Email       string
+}
+
+type Team struct {
+	ID            int
+	DisplayName   string
+	Members       int
+	MemberDetails []TeamMember
```

Comment:
I think it would make more sense to either have a separate `Team` type for what you have done, or remove the `Members` field and rename `MemberDetails->Members`

Reply:
I'll do the remove/rename, since we're getting the same API response back either way